__tests__/nc_news.test.js

```diff
@@ -123,3 +123,218 @@ describe('GET /api/articles/:article_id', () => {
 			});
     });
 });
+
+
+describe('POST, /api/articles/:article_id/comments', () => {
+    it('POST:201 should return correct status code', () => {
+        return request(app).post('/api/articles/1/comments')
+        .send({
+            username: "icellusedkars",
+            body: "Enjoyed it"
+        })
+        .expect(201);
+    });
+    it('POST:201 should post new comment and return comment with confirmation', () => {
+        return request(app).post('/api/articles/1/comments')
+        .send({
+            username: "icellusedkars",
+            body: "Enjoyed it"
+        })
+        .then(({body}) => {
+            expect(body.comment).toHaveProperty("comment_id");
+            expect(body.comment).toHaveProperty("body");
+            expect(body.comment).toHaveProperty("article_id");
+            expect(body.comment).toHaveProperty("author");
+            expect(body.comment).toHaveProperty("votes");
+            expect(body.comment).toHaveProperty("created_at");
+
+        })
+    });
+    it('POST:400 should give error when passed object with incorrect format', () => {
+        return request(app).post('/api/articles/1/comments')
+        .send({
+            name: "icellusedkars",
+            body: "Enjoyed it"
+        })
+        .expect(400)
+        .then(({body}) => {
+            expect(body.msg).toBe("Bad request");
+        })
+    });
+    it('POST:201 should ignore unecessary additional properties', () => {
+        return request(app)
+        .post('/api/articles/1/comments')
+        .send({
+            username: "icellusedkars",
+            body: "Enjoyed it",
+            extra: "Extra thing",
+            extra2: "another one"
+        })
+        .expect(201)
+        .then(({body}) => {
+            expect(body.comment).toHaveProperty("comment_id");
+            expect(body.comment).toHaveProperty("body");
+            expect(body.comment).toHaveProperty("article_id");
+            expect(body.comment).toHaveProperty("author");
+            expect(body.comment).toHaveProperty("votes");
+            expect(body.comment).toHaveProperty("created_at");
+
+        })
+    });
```

Comment:
Good implementation here of the suggestion above. These tests cover all of the keys but not the values. You could make your tests more robust by adding a second argument to check the values, eg
` expect(body.comment).toHaveProperty("body", expect.any(String));`